src/components/templates/PosterHall/components/PosterHallSearch/PosterHallSearch.scss

```diff
@@ -0,0 +1,22 @@
+@import "scss/constants";
+
+.PosterHallSearch {
+  display: flex;
+  width: 100%;
+
+  justify-content: center;
+  margin: $spacing--xxl 0;
+
+  &__input-container {
+    width: 40%;
+    min-width: 300px;
+  }
```

Comment:
I don't think it should have one. It will shrink to `&__input-container` + `checkbox` width, which is what I was trying to achieve.

![image](https://user-images.githubusercontent.com/63194656/117036414-15186600-ad0e-11eb-8d93-ec95eab1f468.png)

Reply:
Does it actually shrink to that size, or do those elements just 'break out' of the container now? I would expect it's probably going to break out? Can't really tell from your screenshot (if you hover over the container element in devtools it should highlight it and show it's bounds/etc that you can then screenshot)